docs/openapi/schemas.yaml

```diff
@@ -1598,3 +1598,313 @@ ApiKeyResponse:
         apiKey:
           description: The API key
           type: string
+OpportunityStatus:
+  description: Status of an opportunity
+  type: string
+  enum:
+    - 'NEW'
+    - 'IN_PROGRESS'
+    - 'IGNORED'
+    - 'RESOLVED'
+  example: 'IN_PROGRESS'
+Opportunity:
+  type: object
+  properties:
+    id:
+      description: UUID of this opportunity
+      $ref: '#/Id'
+    siteId:
+      description: UUID of the site this opportunity is for
+      $ref: '#/Id'
+    auditId:
+      description: UUID of the audit result that triggered this opportunity
+      $ref: '#/Id'
+    runbook:
+      description: URL to the runbook for this opportunity
+      type: string
+      format: url
+      example: 'https://example.com/runbook/broken-backlinks'
+    type:
+      description: Type of opportunity
+      type: string
+      enum:
+        - 'broken-backlinks'
+        - 'broken-internal-links'
+      example: 'broken-backlinks'
+    data:
+      description: Optional map of type-specific data relevant to this opportunity
+      type: object
+      additionalProperties: true
+    origin:
+      description: Origin of opportunity
+      type: string
+      enum:
+        - 'ESS_OPS'
+        - 'AI'
+      example: 'ESS_OPS'
+    title:
+      description: Title of this opportunity
+      type: string
+      example: 'Broken backlinks'
+    description:
+      description: Description of this opportunity
+      type: string
+      example: 'Authoritative domains are linking to invalid SecurBank URLs. This could impact your SEO.'
+    status:
+      description: Status of this opportunity
+      $ref: '#/OpportunityStatus'
+    guidance:
+      description: Guidance text specific to this opportunity
+      type: object
+      additionalProperties: true
+    tags:
+      type: array
+      items:
+        type: string
+      description: Optional list of tags
+      example:
+        - 'Traffic acquisition'
+        - 'Backlinks'
+    createdAt:
+      description: UTC timestamp of when the opportunity was created
+      $ref: '#/DateTime'
+    createdBy:
+      type: string
+      description: IMS or other user ID of who created this opportunity
+      example: '47995190583E98E70A495D53@adobe.com'
+    updatedAt:
+      description: UTC timestamp of when the opportunity was last updated
+      $ref: '#/DateTime'
+    updatedBy:
+      description: IMS or other user ID of who updated this opportunity
+      type: string
+      example: '67994690583E98E70A495D53@adobe.com'
+  required:
+    - id
+    - siteId
+    - auditId
+    - runbook
+    - type
+    - origin
+    - title
+    - status
+    - createdAt
+    - createdBy
```

Comment:
createdBy and updatedBy will not be initially available and be derived automatically from auth context in the future

Reply:
This is the entity for the responses. I let the property still be required, but included this mention in the description of it. Would that be fine or do you want to have it required only when we'll be able to derive it?

Reply:
yes, for clarity purposes and to prevent wrong expectations, i'd leave it out for now.

Reply:
Okay, I removed them.